@osnipso CI has cekit 2.2.1 installed, but we're getting an image build error:

```
cekit build --target target-docker --tag jboss-datagrid-7/datagrid72-openshift:latest
[32m2018-12-03 20:22:10,659 cekit        INFO     Generating files for docker engine.[0m
[32m2018-12-03 20:22:10,753 cekit        INFO     Initializing image descriptor...[0m
[33m2018-12-03 20:22:12,156 cekit        WARNING  The way of defining repository 'jboss-os' is deprecated. Convert it to an URL based repository object. Consult Cekit docs, for more details.[0m
[33m2018-12-03 20:22:12,157 cekit        WARNING  The way of defining repository 'jboss-rhscl' is deprecated. Convert it to an URL based repository object. Consult Cekit docs, for more details.[0m
[33m2018-12-03 20:22:12,179 cekit        WARNING  The way of defining repository 'jboss-os' is deprecated. Convert it to an URL based repository object. Consult Cekit docs, for more details.[0m
[33m2018-12-03 20:22:12,206 cekit        WARNING  The way of defining repository 'jboss-os' is deprecated. Convert it to an URL based repository object. Consult Cekit docs, for more details.[0m
[33m2018-12-03 20:22:12,452 cekit        WARNING  The way of defining repository 'jboss-os' is deprecated. Convert it to an URL based repository object. Consult Cekit docs, for more details.[0m
[33m2018-12-03 20:22:12,471 cekit        WARNING  The way of defining repository 'jboss-os' is deprecated. Convert it to an URL based repository object. Consult Cekit docs, for more details.[0m
[33m2018-12-03 20:22:12,706 cekit        WARNING  The image has ContentSets repositories specified, all other repositories are removed![0m
[31m2018-12-03 20:22:12,715 cekit        ERROR    ODCS is not installed, please install 'odcs-client' package[0m
make: *** [build-image] Error 1
```

As part of these changes do we need to update the build cmd?

--- a/container.yaml
+++ b/container.yaml
@@ -1,0 +1,5 @@
+platforms:
+  only:
+    - x86_64
+compose:
+  pulp_repos: true

--- a/content_sets.yml
+++ b/content_sets.yml
@@ -1,0 +1,3 @@
+x86_64:
+  - rhel-server-rhscl-7-rpms
+  - rhel-7-server-rpms

--- a/image.yaml
+++ b/image.yaml
@@ -202,9 +202,7 @@ modules:
           - name: os-logging
           - name: datagrid.launch
 packages:
-      repositories:
-          - jboss-os
-          - jboss-rhscl
+      content_sets_file: content_sets.yml
       install:
           - rh-mongodb32-mongo-java-driver
           - postgresql-jdbc
@@ -227,6 +225,8 @@ run:
       cmd:
           - "/opt/datagrid/bin/openshift-launch.sh"
 osbs:
+      configuration:
+            container_file: container.yaml
       repository:
             name: containers/jboss-datagrid-7
             branch: jb-datagrid-7.2-openshift-rhel-7

